### ISSUE TITLE
can't use parcel-plugin-vue with parcel-plugin-typescript

### ISSUE THREAD

**Issue by csvwolf** [fathyb/parcel-plugin-typescript#2]
If I use parcel-plugin-vue only like:

```javascript
// index.js
import Vue from 'vue'
import App from './app.vue'
new Vue({
  el: '#app',
  render: h => h(App)
})
```

app.vue:

```vue
<template>
  <section class="app">
    <h1>Hello World</h1>
  </section>
</template>
```

But when I change it to `index.ts`, there is an error:

```
Uncaught TypeError: Cannot read property 'split' of undefined
```

in `version = Vue.version.split('.').map(Number)`

Is there something wrong with my config?

Waiting for response. THX anyway!

The issue is also open in: https://github.com/lc60005457/parcel-plugin-vue/issues/1

**Comment by fathyb** [comment]
I cloned your repo and did not found Parcel in the dependencies. It needs to be in your node_modules for the plugin to work correctly. So I did :

- `yarn add parcel-bundler`
- `node_modules/.bin/parcel index.html`

Then I got this :
![error](https://i.gyazo.com/9483c46b4d9dd878398bcd04ecce57f0.png)

But that's fine because your bundle is still bundled, it's just a reporting. The simplest workaround for this would be to add `declare module '*.vue'` in a `.d.ts` file in your project. I made you a PR showing the changes for building everything correctly.

Note that using this workaround the `.vue` files will be imported as `any` and will not be type-checked. If you want we can add Vue SFC type-checking to the roadmap, we already do it for Angular.
